spec/models/claim_spec.rb

```diff
@@ -363,7 +363,7 @@
   describe "awaiting_approval" do
     let!(:submitted_claims) { create_list(:claim, 5, :submitted) }
     let!(:unsubmitted_claims) { create_list(:claim, 2, :submittable) }
-    let!(:approved_claims) { create_list(:claim, 5, :submitted, approved_at: Time.zone.now) }
+    let!(:approved_claims) { create_list(:claim, 5, :submitted, check: create(:check, status: :approved)) }
```

Comment:
```suggestion
    let!(:approved_claims) { create_list(:claim, 5, :approved) }
```

Reply:
I did think about that, but we thought it was better to be explicit in this case

Reply:
I think it it makes it harder to make changes to the approval process. You have to come and update this spec, when you've already updated the (identically configured) trait.